lib/StripeMethod.js

```diff
@@ -87,6 +87,17 @@ function stripeMethod(spec) {
       }
 
       var requestPath = this.createFullPath(commandPath, urlData);
+      var options = {headers: objectAssign(opts.headers, spec.headers)};
+
+      if (spec.validator) {
+        try {
+          spec.validator(data, options);
+        } catch (err) {
+          reject(err);
+          return;
+        }
+      }
+
```

Comment:
So this seems okay, but I'm always a little wary of breaking out a new global pattern for a new resource, especially when we've already got dozens of resources in here that didn't need it up until now. Have you considered the trade off of just allowing the API to return an error in case `Stripe-Version` wasn't specified in the way that it would if you forgot say, a required parameter? Agreed generally that client-side validation is better where possible for faster feedback, but it seems like a way that we could make ephemeral keys less special cased and avoid new general purpose infrastructure.